### PR TITLE
clean up test_create logging output

### PR DESCRIPTION
**Pull request by kalefranz**
CC @mcg1969

### CODE REVIEW

**Comment by mcg1969** [comment]
Looks great

File: tests/test_create.py [hunk 1]
```diff
@@ -56,41 +65,47 @@ def install_in_env(env_path, *packages):
...
 def test_dash_c_usage_replacing_python():
-    with make_temp_env("-c conda-forge python=3.5") as env_path:
-        assert exists(join(env_path, bindir, 'python3.5'))
-        install_in_env(env_path, "decorator")
-        # TODO @mcg1969: now what?
+    with disable_dotlog():
+        with make_temp_env("-c conda-forge python=3.5") as env_path:
+            assert exists(join(env_path, bindir, 'python3.5'))
+            install_in_env(env_path, "decorator")
+            # TODO @mcg1969: now what?
```
Comment (mcg1969) [inline tests/test_create.py]: If we're testing against this regression we would need to make sure that conda-forge Python is still present. So we'd have to inspect `conda-meta/python-*.json` to make sure it has `conda-forge` in the `url` field.

Comment (mcg1969) [inline tests/test_create.py]: Maybe `any(p.startswith('conda-forge::python-3.5' for p in conda.install.linked()))`

**Comment by kalefranz** [comment]
@mcg1969 The line here should pass now, correct?  I've rebased this branch against master.
https://github.com/conda/conda/pull/2620/files#diff-ae879015ffad709a9a28c1468d6cf71cR113

**Comment by mcg1969** [comment]
That's a good way to do it.

**Comment by kalefranz** [comment]
I'm merging this to master because it *should* pass in master now.  If it doesn't, we'll find out why there.

**Pull request merged**